src/lib/ciul/pd.c

```diff
@@ -19,22 +19,47 @@
 
 #include <net/if.h>
 
+static int __ef_tok_eq(const char* tok, size_t tok_len, const char* tmpl)
+{
+    size_t l = strlen(tmpl);
+    return l == tok_len && ! strncmp(tok, tmpl, l);
+}
+
+static enum ef_pd_flags __ef_env2flags(enum ef_pd_flags flags)
+{
+    const char* s = getenv("EF_VI_PD_FLAGS");
+    if( s == NULL )
+        return flags;
+
+    enum ef_pd_flags new_flags = 0;
+    const char* tok_end;
+    do
+    {
+        tok_end = strchr(s, ',');
+        if( ! tok_end )
+            tok_end = s + strlen(s);
+        if( __ef_tok_eq(s, tok_end - s, "vf") )
+            new_flags |= EF_PD_VF;
+        else if( __ef_tok_eq(s, tok_end - s, "phys") )
+            new_flags |= EF_PD_PHYS_MODE;
+        else if( __ef_tok_eq(s, tok_end - s, "default") )
+            flags = 0;
+        else if( __ef_tok_eq(s, tok_end - s, "mcast_loop") )
+            new_flags |= EF_PD_MCAST_LOOP;
+        s = tok_end + 1;
+    } while( *tok_end != '\0' );
+
+    return new_flags != 0 ? new_flags : flags;
```

Comment:
I think this has changed the behaviour of `EF_VI_PD_FLAGS=default`

Reply:
Not really (unless I'm missing something).
Stand-alone `default` resets the `flags` to zero (not `new_flags`).

Reply:
Oh yes, so it does.